### PR TITLE
FPAY-899: Added redis for bank-account-tokenisation service

### PR DESCRIPTION
**Pull request by tginszt**
Added bank-account-tokenisation service to shared UW redis config

### CODE REVIEW

File: redis/Makefile [hunk 1]
```diff
@@ -16,3 +16,18 @@ gen-dev-enablement-redis-shared:
...
 		alpine/helm ./gen-yaml/gen.sh
+
+.PHONY: gen-bank-account-tokenisation-redis-shared
+gen-bank-account-tokenisation-redis-shared:
+	docker run -ti --rm \
+		--volume $${PWD}:/opt/manifests \
+		--env BITNAMI_REDIS_RELEASE=${BITNAMI_REDIS_RELEASE} \
+		--env NS="payment-platform-banking" \
+		--env NAME="bank-account-tokenisation-redis" \
+		--env OPSLEVEL_APP_DESCRIPTION="cache for opslevel-k8s-deployer" \
```
Comment (MarcinGinszt) [inline redis/Makefile:27]: It needs description suiting your service :slightly_smiling_face:

Comment (tginszt) [reply redis/Makefile]: Updated the description.

Comment (tginszt) [reply redis/Makefile]: Need to regenerate, wait a sec

Comment (tginszt) [reply redis/Makefile]: Ok, should be all good now : )

Comment (MarcinGinszt) [reply redis/Makefile]: LGTM, but I'm afraid you'll have to regenerate it anyway when https://github.com/utilitywarehouse/shared-kustomize-bases/pull/6 will be merged.

**Review by MarcinGinszt — APPROVED** [review]

File: redis/Makefile (hunk 1 above)
Comment (tginszt) [reply redis/Makefile]: Will keep an eye out for that

**Pull request merged**